device/alpaka/CMakeLists.txt

```diff
@@ -7,37 +7,31 @@
 # Project include(s).
 include( traccc-compiler-options-cpp )
 
+set(PUBLIC_LIBRARIES traccc::core detray::core detray::utils vecmem::core covfie::core)
+set(PRIVATE_LIBRARIES traccc::device_common alpaka::alpaka)
+
 if(alpaka_ACC_GPU_CUDA_ENABLE)
   enable_language(CUDA)
-  find_package( CUDAToolkit REQUIRED )
-
   include( traccc-compiler-options-cuda )
 
-  set_source_files_properties(src/utils/make_prefix_sum_buff.cpp PROPERTIES LANGUAGE CUDA)
-  set_source_files_properties(src/seeding/spacepoint_binning.cpp PROPERTIES LANGUAGE CUDA)
-elseif(alpaka_ACC_GPU_HIP_ENABLE)
```

Comment:
Will have to put some HIP logic back in but easier to get this merged first

Reply:
Yeah, https://github.com/alpaka-group/alpaka/blob/develop/cmake/addLibrary.cmake deals with some parts, but its possible we will need some extra bits here.

I'm not actually sure how needed the `enable_language(CUDA)` bit is, since that is also in the Alpaka wrapper. Its possible I needed it previously, but since fixing the issue with the examples going via `nvcc`, it may not be needed now.